### ISSUE TITLE
Realtime notification

### ISSUE THREAD

**Issue by neronmoon** [parautenbach/TeamCity-HipChat-Notifier#70]
TeamCity can mark build as failed before it ended. Need to send notification when it happens.

https://confluence.jetbrains.com/display/TCD9/Build+Script+Interaction+with+TeamCity

**Comment by neronmoon** [comment]
Any thoughts here?

**Comment by parautenbach** [comment]
Yes, TC supports a "build failing" event and it can be implemented. I have very limited time currently, but this is open source, so I'll accept PRs.
